state-compare/src/main.rs

```diff
@@ -534,14 +572,19 @@ async fn compare_active_bakers(
     client2: &mut v2::Client,
     block1: BlockHash,
     block2: BlockHash,
+    pv1: ProtocolVersion,
+    pv2: ProtocolVersion,
 ) -> anyhow::Result<bool> {
     println!("Checking active bakers.");
     let mut found_diff = false;
     let (ei1, ei2) = futures::try_join!(
         client1.get_election_info(block1),
         client2.get_election_info(block2)
     )?;
-    if ei1.response.election_difficulty != ei2.response.election_difficulty {
+    if pv1 < ProtocolVersion::P6
```

Comment:
```suggestion
    if pv2 < ProtocolVersion::P6
```

Reply:
I am checking that both pv1 and pv2 are below P6. I suppose I could just check pv2 and not both.

Or am I missing something?

Reply:
Discussed offline, will keep it as is.